app/models/content_page.rb

```diff
@@ -18,4 +18,11 @@ def body
   def image
     translate(:image)
   end
+
+  def translate(label)
```

Comment:
This method has slipped back in from a previous iteration of the code. The `translate` method is now provided by the shared concern `TranslateFromLocale`